tests/test_backend.py

```diff
@@ -120,7 +121,10 @@ def test_client_delete(get_cluster_info):
 
     backend = ElastiPymemcache('h:0', {})
     ret = backend.delete('key1')
-    eq_(ret, None)
+    if django.get_version() >= '3.1':
```

Comment:
See: https://docs.djangoproject.com/en/3.1/releases/3.1/#cache

> delete() now returns True if the key was successfully deleted, False otherwise.

Why `delete_many` still returned `None` 🤔  ??